### ISSUE TITLE
[Bug] An error occurred while extracting the Universal Packages tooling: [WinError 5] Access is denied

### ISSUE THREAD

**Issue by 962720900** [Azure/azure-devops-cli-extension#914]
**Describe the bug**

I want to publish a universal artifact to Azure DevOps

1. login using az login
2. setup defaults..
3. az artifacts universal publish --feed SIT --name docker --version 0.1 --path .
- Downloading Universal Packages tooling (ArtifactTool_win10-x64_0.2.128): 99.92% ..

An error occurred while extracting the Universal Packages tooling: [WinError 5] Access is denied: 'C:\\Users\\WS19312\\.azure\\azuredevops\\cli\\tools\\artifacttool\\a445ec82-2166-4964-946e-d868ead0c4e3' -> 'C:\\Users\\WS19312\\.azure\\azuredevops\\cli\\tools\\artifacttool\\ArtifactTool_win10-x64_0.2.104'
[WinError 2] The system cannot find the file specified
Traceback (most recent call last):
pip-install-76yvurxi\knack\knack\cli.py, ln 206, in invoke
cli\core\commands\__init__.py, ln 575, in execute
cli\core\commands\__init__.py, ln 633, in _run_jobs_serially
cli\core\commands\__init__.py, ln 624, in _run_job

We have seen your team already put the bug to your backlog, may I know the estimate & the dateline you will fix the issue?

Thanks for you help in advance.

**Comment by satbai** [comment]
Could you run az --version and share the output please? This will help us see which versions you're using. I recommend updating to the latest azure-cli version, the latest version of azure-devops extension and the latest python version. Also, which OS are you on?

**Comment by 962720900** [comment]
Hi @satbai  Thanks for your comments, the details of the versions are shown below,
azure-cli                         2.0.77
command-modules-nspkg              2.0.3
core                              2.0.77
nspkg                              3.0.4
telemetry                          1.0.4

Extensions:
azure-devops                      0.12.0

If there need more details, please feel free let me know.

**Comment by satbai** [comment]
There is a newer version of the azure-devops available. You can update it with `az extension update --name azure-devops`. I don't know which version of python you're using but make sure you're using python3 as well. Once these steps are done, do you still get the same error?

**Comment by shubham90** [comment]
This bug occurs when some other process might interfere with the download artifact tool. The process could be an antivirus, some other process which is sharing the same location etc. We are in the process of adding a retry to fix this. 

Also, this issue looks like a duplicate of #798 . Closing this one and tracking it there.

**Comment by 962720900** [comment]
@shubham90 I have checked that the the latest release of az devops cli extention is [0.17.0](https://github.com/Azure/azure-devops-cli-extension/releases/tag/20200113.1).
I execute the update command : az extension update --name azure-devops  
The az devops extention cli version still is 0.16.0.
![image](https://user-images.githubusercontent.com/27622845/72404892-edf0fa80-3791-11ea-8913-1a44bc3856cd.png)
Could you please help to this?

**Comment by ishitam8** [comment]
@962720900 , New release is still in review  from Azure CLI front . [Refer this PR.](https://github.com/Azure/azure-cli-extensions/pull/1195)
Once this PR is merged, you will be able to update Devops CLI to 0.17.0

**Comment by 962720900** [comment]
@ishitam8 Thx
